{"uid":"8887f627613678a7","name":"Тест 1","fullName":"test#test_1","historyId":"481c50c2a0ce3c4af0e5e11c30040d79","time":{"start":1679240466646,"stop":1679240466646,"duration":0},"description":"Тест 1","descriptionHtml":"<p>Тест 1</p>\n","status":"broken","statusMessage":"ValueError: There is no such driver by url https://chromedriver.storage.googleapis.com/111.0.5547/chromedriver_linux64.zip","statusTrace":"@pytest.fixture(scope=\"function\")\n    def driver():\n        print(\"\\nbrowser open\")\n        link = \"https://www.dns-shop.ru/\"\n>       driver = webdriver.Chrome(service=ChromiumService(ChromeDriverManager(chrome_type=ChromeType.CHROMIUM).install()))\n\nconftest.py:29: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.8.16/x64/lib/python3.8/site-packages/webdriver_manager/chrome.py:39: in install\n    driver_path = self._get_driver_path(self.driver)\n/opt/hostedtoolcache/Python/3.8.16/x64/lib/python3.8/site-packages/webdriver_manager/core/manager.py:30: in _get_driver_path\n    file = self._download_manager.download_file(driver.get_url())\n/opt/hostedtoolcache/Python/3.8.16/x64/lib/python3.8/site-packages/webdriver_manager/core/download_manager.py:28: in download_file\n    response = self._http_client.get(url)\n/opt/hostedtoolcache/Python/3.8.16/x64/lib/python3.8/site-packages/webdriver_manager/core/http.py:33: in get\n    self.validate_response(resp)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nresp = <Response [404]>\n\n    @staticmethod\n    def validate_response(resp: requests.Response):\n        status_code = resp.status_code\n        if status_code == 404:\n>           raise ValueError(f\"There is no such driver by url {resp.url}\")\nE           ValueError: There is no such driver by url https://chromedriver.storage.googleapis.com/111.0.5547/chromedriver_linux64.zip\n\n/opt/hostedtoolcache/Python/3.8.16/x64/lib/python3.8/site-packages/webdriver_manager/core/http.py:16: ValueError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"driver","time":{"start":1679240466647,"stop":1679240468239,"duration":1592},"status":"broken","statusMessage":"ValueError: There is no such driver by url https://chromedriver.storage.googleapis.com/111.0.5547/chromedriver_linux64.zip\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.8.16/x64/lib/python3.8/site-packages/pluggy/_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"/opt/hostedtoolcache/Python/3.8.16/x64/lib/python3.8/site-packages/_pytest/fixtures.py\", line 1128, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/opt/hostedtoolcache/Python/3.8.16/x64/lib/python3.8/site-packages/_pytest/fixtures.py\", line 900, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/runner/work/DNS_shop_autotests/DNS_shop_autotests/conftest.py\", line 29, in driver\n    driver = webdriver.Chrome(service=ChromiumService(ChromeDriverManager(chrome_type=ChromeType.CHROMIUM).install()))\n  File \"/opt/hostedtoolcache/Python/3.8.16/x64/lib/python3.8/site-packages/webdriver_manager/chrome.py\", line 39, in install\n    driver_path = self._get_driver_path(self.driver)\n  File \"/opt/hostedtoolcache/Python/3.8.16/x64/lib/python3.8/site-packages/webdriver_manager/core/manager.py\", line 30, in _get_driver_path\n    file = self._download_manager.download_file(driver.get_url())\n  File \"/opt/hostedtoolcache/Python/3.8.16/x64/lib/python3.8/site-packages/webdriver_manager/core/download_manager.py\", line 28, in download_file\n    response = self._http_client.get(url)\n  File \"/opt/hostedtoolcache/Python/3.8.16/x64/lib/python3.8/site-packages/webdriver_manager/core/http.py\", line 33, in get\n    self.validate_response(resp)\n  File \"/opt/hostedtoolcache/Python/3.8.16/x64/lib/python3.8/site-packages/webdriver_manager/core/http.py\", line 16, in validate_response\n    raise ValueError(f\"There is no such driver by url {resp.url}\")\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":true,"hasContent":true}],"testStage":{"description":"Тест 1","status":"broken","statusMessage":"ValueError: There is no such driver by url https://chromedriver.storage.googleapis.com/111.0.5547/chromedriver_linux64.zip","statusTrace":"@pytest.fixture(scope=\"function\")\n    def driver():\n        print(\"\\nbrowser open\")\n        link = \"https://www.dns-shop.ru/\"\n>       driver = webdriver.Chrome(service=ChromiumService(ChromeDriverManager(chrome_type=ChromeType.CHROMIUM).install()))\n\nconftest.py:29: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.8.16/x64/lib/python3.8/site-packages/webdriver_manager/chrome.py:39: in install\n    driver_path = self._get_driver_path(self.driver)\n/opt/hostedtoolcache/Python/3.8.16/x64/lib/python3.8/site-packages/webdriver_manager/core/manager.py:30: in _get_driver_path\n    file = self._download_manager.download_file(driver.get_url())\n/opt/hostedtoolcache/Python/3.8.16/x64/lib/python3.8/site-packages/webdriver_manager/core/download_manager.py:28: in download_file\n    response = self._http_client.get(url)\n/opt/hostedtoolcache/Python/3.8.16/x64/lib/python3.8/site-packages/webdriver_manager/core/http.py:33: in get\n    self.validate_response(resp)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nresp = <Response [404]>\n\n    @staticmethod\n    def validate_response(resp: requests.Response):\n        status_code = resp.status_code\n        if status_code == 404:\n>           raise ValueError(f\"There is no such driver by url {resp.url}\")\nE           ValueError: There is no such driver by url https://chromedriver.storage.googleapis.com/111.0.5547/chromedriver_linux64.zip\n\n/opt/hostedtoolcache/Python/3.8.16/x64/lib/python3.8/site-packages/webdriver_manager/core/http.py:16: ValueError","steps":[],"attachments":[{"uid":"98f675b6ae254da8","name":"stdout","source":"98f675b6ae254da8.txt","type":"text/plain","size":14}],"parameters":[],"stepsCount":0,"attachmentsCount":1,"shouldDisplayMessage":true,"hasContent":true},"afterStages":[],"labels":[{"name":"tag","value":"flaky(reruns=2, reruns_delay=5)"},{"name":"suite","value":"test"},{"name":"host","value":"fv-az581-383"},{"name":"thread","value":"2443-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"test"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":true,"retry":true,"extra":{"categories":[],"tags":["flaky(reruns=2, reruns_delay=5)"]},"source":"8887f627613678a7.json","parameterValues":[]}